_toc.yml

```
@@ -0,0 +1,61 @@
+format: jb-book
+root: README
+parts:
+  - caption: Getting started
+    chapters:
+    - file: notebooks/START_HERE
```

I don't think this notebook exists, maybe we can change it to get started doc?
```suggestion
    - file: docs/get-started
```